### PR TITLE
add minitox to integrations

### PR DESCRIPTION
**Pull request by hqwrong**
see https://github.com/TokTok/c-toxcore/pull/874

---
This change is [Reviewable](https://reviewable.io/reviews/toktok/website/122)

### CODE REVIEW

File: toktok/integrations.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ yourself to this page.
 *   [Toxy](https://github.com/alexbakker/Toxy)
 *   [Toxzilla](https://github.com/toxzilla/app)
 *   [uTox](https://github.com/uTox/uTox)
+*   [minitox](https://github.com/hqwrong/minitox)
```
Comment (iphydf) [inline toktok/integrations.md]: Can you keep this list sorted?

Comment (hqwrong) [reply toktok/integrations.md]: Sure

**Comment by iphydf** [comment]
:lgtm_strong:

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io/reviews/toktok/website/122#-:-LEGuvry2ol45Vs4uS9-:b-jbsf8z)*

**Comment by iphydf** [comment]
Please enable the checkbox "Allow edits from maintainers." on the bottom right.

**Comment by hqwrong** [comment]
@iphydf I find it was already enabled.

**Pull request merged**